appengine/sendgrid/README.md

```diff
@@ -0,0 +1,29 @@
+# Ruby SendGrid email sample for Google App Engine
+
+This sample demonstrates how to use [SendGrid](https://www.sendgrid.com) on
+[Google App Engine Managed VMs](https://cloud.google.com/appengine).
+
+For more information about SendGrid, see their
+[documentation](https://sendgrid.com/docs/User_Guide/index.html).
+
+## Setup
+
+Before you can run or deploy the sample, you will need to do the following:
+
+1. [Create a SendGrid Account](http://sendgrid.com/partner/google). As of
```

Comment:
I think this partnership ended. Now SendGrid offers 12,000 free emails for everyone, also linked page doesn't state anything about 25k free e-mails.